soroban-sdk/src/env.rs

```diff
@@ -582,9 +582,18 @@ impl Env {
     /// with that contract ID. Providing `None` causes the Env to generate a new
     /// contract ID that is assigned to the contract.
     ///
+    /// If a contract has a constructor defined, then it will be called with
+    /// no arguments. In case if constructor has to take some arguments, use
+    /// `register_contract_with_constructor`.
```

Comment:
Suggested trimming to remove superfluous words:
```suggestion
    /// no arguments. If a constructor takes arguments, use
    /// `register_contract_with_constructor`.
```

Reply:
Updated.